static/styles/settings.css

```diff
@@ -1683,6 +1683,21 @@ body:not(.night-mode) #settings_page .custom_user_field .datepicker {
     margin-left: 2px;
 }
 
+.new-style .btn-disabled {
+    color: hsl(208, 7%, 46%);
+    border: 1px solid hsl(208, 7%, 46%);
+}
+
+.new-style .btn-disabled:hover {
+    color: hsl(0, 0%, 100%);
+    background-color: hsl(208, 7%, 46%);
+}
+
+.new-style .btn-danger:hover {
+    color: hsl(0, 0%, 100%);
+    background-color: hsl(354, 70%, 54%);
+}
```

Comment:
@ligmitz since this is not normal css but postcss, we can take advantage of the nesting. Line 31 starts the nesting for .new-style. You can club these classes under it and try using `&:hover` syntax of scss too.